lib/rubycritic/platforms/darwin.rb

```diff
@@ -0,0 +1,26 @@
+require "rubycritic/platforms/base"
+
+module Rubycritic
+  module Platforms
+    class Darwin < Base
+      BROWSER_MAP = {}.tap do |hash|
+        hash.merge! :chrome => "Google Chrome", :firefox => "firefox", :safari => "safari"
+        hash.default = "safari"
+      end
+
+      def can_open?
+        platform == :darwin
+      end
+
+      def application
+        BROWSER_MAP[browser_name]
+      end
+      alias_method :browser, :application
```

Comment:
This will also break the build after rebase because rubocop will complain about that, sorry ;)
Should be `alias`.